src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Utils/InternalMetadataProviderIdentityExtensions.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Sbom.Parsers.Spdx22SbomParser.Utils
     /// <summary>
     /// Provides helper functions to generate identity strings for SPDX.
     /// </summary>
-    public class IdentityUtils
+    public static class InternalMetadataProviderIdentityExtensions
```

Comment:
wont this make it hard to mock this class?
whats your reasoning behind making this an extension class?

Reply:
This suggestion - https://github.com/microsoft/sbom-tool/pull/92#discussion_r939105511 .  I don't think it will make testing harder, but @ksigmund you have any thoughts on that?